examples/fungible/web-frontend/src/App.js

```diff
@@ -61,7 +61,7 @@ function App({ owner }) {
     { data: balanceData, called: balanceCalled, error: balanceError },
   ] = useLazyQuery(GET_BALANCE, {
     fetchPolicy: "network-only",
-    variables: { owner: { User: owner } },
+    variables: { owner: `User:${owner}` },
```

Comment:
curious what is happening here :)

Reply:
I believe the schema for `AccountOwner` changed (maybe in [6a6c89b](https://github.com/linera-io/linera-protocol/commit/6a6c89b4d83d66fa32782299bb9c6df492142fd8)?) but the web frontend was still using the old schema.